doc/howtos/rdtraining/basicmodel.rst

```diff
@@ -0,0 +1,294 @@
+
+        name = fields.Char()
+
+The ``name`` field is a :class:`~odoo.fields.Char` which will be represented as a Python
+``unicode`` and a SQL ``VARCHAR``.
+
+Types
+-----
+
+.. note::
+
+    **Goal**: at the end of this section, several basic fields are added to the table
+    ``estate_property``:
+
+    .. code-block:: text
+
+        $ psql -d rd-demo
+
+        rd-demo=# \d estate_property;
+                                                    Table "public.estate_property"
+            Column       |            Type             | Collation | Nullable |                   Default
+        --------------------+-----------------------------+-----------+----------+---------------------------------------------
+        id                 | integer                     |           | not null | nextval('estate_property_id_seq'::regclass)
+        create_uid         | integer                     |           |          |
+        create_date        | timestamp without time zone |           |          |
+        write_uid          | integer                     |           |          |
+        write_date         | timestamp without time zone |           |          |
+        name               | character varying           |           |          |
+        description        | text                        |           |          |
+        postcode           | character varying           |           |          |
+        date_availability  | date                        |           |          |
+        expected_price     | double precision            |           |          |
+        selling_price      | double precision            |           |          |
+        bedrooms           | integer                     |           |          |
+        living_area        | integer                     |           |          |
+        facades            | integer                     |           |          |
+        garage             | boolean                     |           |          |
+        garden             | boolean                     |           |          |
+        garden_area        | integer                     |           |          |
+        garden_orientation | character varying           |           |          |
+        Indexes:
+            "estate_property_pkey" PRIMARY KEY, btree (id)
+        Foreign-key constraints:
+            "estate_property_create_uid_fkey" FOREIGN KEY (create_uid) REFERENCES res_users(id) ON DELETE SET NULL
+            "estate_property_write_uid_fkey" FOREIGN KEY (write_uid) REFERENCES res_users(id) ON DELETE SET NULL
+
+
+There are two broad categories of fields: 'simple' fields which are atomic
+values stored directly in the model's table and 'relational' fields linking
+records (of the same model or of different models).
+
+Example of simple fields are :class:`~odoo.fields.Boolean`,
+:class:`~odoo.fields.Float`, :class:`~odoo.fields.Char`,
+:class:`~odoo.fields.Date` or :class:`~odoo.fields.Selection`.
```

Comment:
As Text is used in the example, maybe is a good idea tu put it here (and maybe put a small note to say the difference between Text and Char)